<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunDevice</class>
 <widget class="QDialog" name="RunDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Connected Devices</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="mDevicesList"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="mSameCheckBox">
       <property name="text">
        <string>Use same selection for future launches</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mRefreshBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../ART.qrc">
         <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ART.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RunDevice</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RunDevice</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mDevicesList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>RunDevice</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
